app/helpers/metadata_helper.rb

```diff
@@ -270,8 +289,10 @@ def validate_metadata_csv_for_samples(samples, metadata, new_samples = false)
       end
 
       if extract_host_genome_from_metadata
-        host_genome = host_genomes.select { |cur_host_genome| cur_host_genome.name == row[host_genome_index] }.first
+        host_genome = host_genomes.select { |cur_hg| cur_hg && cur_hg.name.casecmp?(row[host_genome_index]) }.first
 
+        # TODO: (gdingle): This behavior will change after removal of admin-only of new host genome input.
```

Comment:
add date?

Reply:
I think the link to the JIRA is sufficient.